### ISSUE TITLE
Stuck in Ebon Hold as a Ghost with corpse in Blades Edge Mountains

### ISSUE THREAD

**Issue by Amplefere** [dalaranwow/dalaran-wow#4892]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
DK Ghost stuck in EBON HOLD with no way out to corpse in Blades Edge Mountains and no way to get to a Spirit Healer. Attempted Character Unstuck with no luck, tried relogging, switching toons, etc. Have had in game ticket open for over an hour with no response
Character name is Dakrfury

**Expected behaviour**: Tell us what should happen instead
Should be able to find a way to Rez

**Steps to reproduce the problem**:

1. Log in to my DK 
2. try to find a way to Rez
3. 

**Include proofs for this behaviour**
http://www.dalaran-wow.com/forums/input-and-assistance/community-support/topic/9244/stuck-in-the-ebon-hold-as-a-ghost

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by TeridanDW** [comment]
The issuetracker is for serverside and gamemechanic issues only. Please write an Ingame-issue to a Gamemaster and the Gamemaster will then help you as soon as he can. (Or create a new Ingame-Account in order to be able to talk with a gamemaster with you new level 1 character) Thank you.

![GitHub Logo](http://i.imgur.com/bYkYeGQ.png)

**Comment by Amplefere** [comment]
The ingame report was never responded to over a period of nearly 4 days.
Was able to correct the issue by having a rezzer in party at the location
of my despawned corpse and rez via party target. I believe this issue to
have occurred because of my attempt to use Character Unstuck as i was
unable to get close enough to my corpse to rez since i could not fly as a
ghost. I believe that it's possible to adjust this for outlands at least
for Blades Edge, Shadowmoon Valley and Netherstrom. While i may not need it
again it would potentially save a lot of future reports as the server
becomes more populated.
